cmd/flag/correct_order_flags.go

```diff
@@ -45,11 +45,11 @@ func (f *CorrectOrderFlags) AddFlags(cmd *cobra.Command) {
 	cmd.Flags().DurationVarP(&f.DelayThreshold, "delay-threshold", "", defaultDelayThreshold,
 		"The maximum amount of time that packets remain in the sorting pool.")
 	cmd.Flags().IntVarP(&f.BundleCountThreshold, "count-threshold", "", defaultBundleCountThreshold,
-		"The maximum number of packets that will be sorted.")
+		"The maximum number of packets in the sorting pool.")
 	cmd.Flags().IntVarP(&f.BundleByteThreshold, "byte-threshold", "", defaultBundleByteThreshold,
-		"The maximum number of bytes of packets that will be sorted.")
+		"The maximum size of the sorting pool in bytes.")
 	cmd.Flags().BoolVarP(&f.CorrectOrder, "correct-order", "", defaultCorrectOrder,
-		"Reordering packets by time_first_byte_received when set to true.")
+		"When set to true, packets will be reordered by time_first_byte_received.")
```

Comment:
Maybe it is better to say "sorted" here instead of "reordered" so it matches the other ones. (Sorry, I just thought of this.)

Reply:
Also, maybe the flag should be called `sort`.  What do you think?

Reply:
`--sort` is obvious and will be simpler for this usage, but I am afraid people might want to use `--sort` option for printing commands such as `list-plans`.

Reply:
Yes, that makes sense.